visidata/loaders/parquet.py

```diff
@@ -16,7 +16,7 @@ def iterload(self):
         pq = vd.importExternal('pyarrow.parquet', 'pyarrow')
         from visidata.loaders.arrow import arrow_to_vdtype
 
-        self.tbl = pq.read_table(self.source)
+        self.tbl = pq.read_table(str(self.source))
```

Comment:
This may remove the ability to use e.g. parquet files inside .zip files.  (Not sure if this works at present either though--depends on if read_table can take a file object).

Reply:
Hmm, good call. Just tested locally and that seems to work properly with or without this change :+1:

Reply:
Huh!  How does *that* work? :)  But if it does, I believe you, and we'll merge it.

Reply:
:clap:

Reply:
> Huh! How does _that_ work? :) But if it does, I believe you, and we'll merge it.

I didn't like how the answer to this was "I have noooo idea" so I started following it and realized I was wrong. It's broken in both cases!

...but my test setup made it look like it worked. Because I just added a parquet file to a zip in the same directory:

```
├── test.zip
│   ├── benchmark.parquet
benchmark.parquet
```

So when I ran `vd` and opened the zip file, then opened the parquet file inside it, it worked! But under the covers it was transforming the source path into the local path _outside_ the zip. If I moved the uncompressed `benchmark.parquet` file elsewhere, things stopped working (with or without this change). Doh.

Reply:
Hah, okay.  Thanks for re-checking.  In any event, this change doesn't break existing behavior, so when I want to revisit my `.parquetz` file idea (in order to have multiple parquet tables in a single file), I'll see if I can fix it without breaking this use case.

Reply:
For the record, opening this as a separate issue https://github.com/saulpw/visidata/issues/1916

I'll try to take a look at it since I'm halfway in the hole already, but if anyone else sorts it out first that works too :)